@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/performance/fn_perfData.sqf

```diff
@@ -0,0 +1,84 @@
+/* ----------------------------------------------------------------------------
+Function: fza_sfmplus_fnc_perfData
+
+Description:
+	Sets the initial mass of the helicopter.
+
+Parameters:
+	_heli - The helicopter to get information from [Unit].
+
+Returns:
+	...
+
+Examples:
+	...
+
+Author:
+	BradMick
+---------------------------------------------------------------------------- */
+params ["_heli"];
+
+private _PA     = 0;  //feet
+private _temp   = 20; //deg C
+private _config = configFile >> "CfgVehicles" >> typeof _heli >> "Fza_SfmPlus";
+
+//Max TQ DE and SE
+private _intMaxTQTable  = [getArray (_config >> "maxTqTable"), _PA] call fza_fnc_linearInterp;
+private _intMaxTQTable2 = [[-40, _intMaxTQTable select 1, _intMaxTQTable select 2],
+						   [-20, _intMaxTQTable select 3, _intMaxTQTable select 4],
+						   [  0, _intMaxTQTable select 5, _intMaxTQTable select 6],
+						   [ 20, _intMaxTQTable select 7, _intMaxTQTable select 8],
+						   [ 40, _intMaxTQTable select 9, _intMaxTQTable select 10]];
+private _intMaxTQTable3 = [_intMaxTQTable2, _temp] call fza_fnc_linearInterp;
+private _maxTQ_DE       = _intMaxTQTable3 select 1;
+private _maxTQ_SE       = _intMaxTQTable3 select 2;
+_heli setVariable ["fza_sfmplus_maxTQ_DE", _maxTQ_DE];
```

Comment:
There's no real need to split these out if they're already in an array. 

I would recommend putting these into a single `"fza_sfmplus_maxTQ_DE"` variable instead.